yt/frontends/boxlib/_deprecation.py

```diff
@@ -0,0 +1,13 @@
+from yt._maintenance.deprecation import issue_deprecation_warning, warnings
+
+
+def boxlib_deprecation():
+    with warnings.catch_warnings():
+        warnings.simplefilter("default")
```

Comment:
I'd assume this pattern would be used to ensure that the warning is raised every time a deprecated module is imported from, but then I don't think that's what `"default"` does. Maybe you meant `"always"` ?

Reply:
@neutrinoceros That sounds good.